src/services/cart.js

```diff
@@ -24,7 +24,7 @@ export default function Cart(oldCart) {
       totalQty += item.quantity;
       totalPrice += item.quantity * item.price;
     });
-    this.totalPrice = totalPrice;
+    this.totalPrice = (totalPrice).toFixed(2);
```

Comment:
Maybe not for this PR, but I think a currency helper would be good. e.g. `{{currency totalPrice}}` ➡️ `$12.34`. Tihs would allow us to drop in different currency formatting options later.

Reply:
Yeah — good idea.